src/components/small-warning/styles/SSmallWarning.tsx

```diff
@@ -0,0 +1,31 @@
+import styled, { css } from 'styled-components';
+import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from '../../../constants/styles/mediaquerys';
+
+const Bronze = css`
+  color: var(--ui-supportive-01);
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  svg{
```

Comment:
@steiningermona please remove this width and let the size be defined by the icon itself.